test/test_helper.rb

```diff
@@ -3,10 +3,10 @@
 require 'rails/test_help'
 require 'minitest/emoji'
 require 'capybara/rails'
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.